ios/RNCallKeep/RNCallKeep.m

```diff
@@ -41,6 +41,7 @@ @implementation RNCallKeep
     BOOL _isStartCallActionEventListenerAdded;
     bool _hasListeners;
     NSMutableArray *_delayedEvents;
+    bool _isSetup;
```

Comment:
Track if we've already called setup.

I assume it's not an existing feature to change the setup of this library after it's initally been configured (?)